Week2/project/index.js

```diff
@@ -7,4 +7,118 @@
   Display minutes and seconds
   If the timer finishes the timer should be replaced by the message: Time 's up!
  * 
- */
\ No newline at end of file
+ */
+ const inputInterval = document.getElementById('inputTime');
+ const minutes = document.getElementById('minutes');
+ const seconds = document.getElementById('seconds');
+ const column = document.getElementById('column');
+ const intervalDown = document.querySelector('.fa-arrow-down');
+ const intervalUp = document.querySelector('.fa-arrow-up');
+ const playBtn = document.querySelector('.fa-play');
+ const pauseBtn = document.querySelector('.fa-pause');
+ const stopBtn = document.querySelector('.fa-stop');
+ const buttons = document.querySelectorAll('i');
+
+ //duration in seconds
+ let duration = 1500;
```

Comment:
I appreciate the comment, perhaps it would have been easier to name the variable differently
```suggestion
 let durationSeconds = 1500;
```